Master-Game/Managers/RenderingManager.cs

```diff
@@ -54,15 +54,18 @@ protected void RenderMap(ref PlayerEntity player)
                     {
                         if(tile.Type == TileType.Grass)
                         {
-                            Console.Write(" ^ ");
+                            //Console.Write(" ^ ");
```

Comment:
Not sure why these needed changing

Reply:
Same for below

Reply:
Yeah, we had decided on these symbols as they kind of look like what they are portraying

Reply:
ok you can keep them but remove the commented ones.. Don't leave commented code in, unless it's to explain something